### PR TITLE
Update Labs examples with ConfigMaps

### PR DESCRIPTION
**Pull request by enriquebelarte**
Example YAML files have been updated in order to match the new way
of setting the build of modules using a ConfigMap as a build parameter in
the declaration of Module objects.

### CODE REVIEW

File: docs/labs/gke/build-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+dockerfile: |
```
Comment (ybettan) [inline docs/labs/gke/build-configmap.yaml:1]: Aren't we missing
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: build-module
data:
```
here?

Comment (enriquebelarte) [reply docs/labs/gke/build-configmap.yaml]: Absolutely. Fixed, thank you!

File: docs/labs/azure/kmm-kmod.yaml [hunk 2]
```diff
@@ -9,39 +9,11 @@ spec:
         moduleName: kmm-ci-a 
 
       kernelMappings:
-        - literal: 5.4.0-1089-azure 
+        - regexp: '^.+.azure'
```
Comment (qbarrand) [inline docs/labs/azure/kmm-kmod.yaml:12]:
```suggestion
        - regexp: '^.+-azure$'
```

File: docs/labs/aws/kmm-kmod.yaml [hunk 3]
```diff
@@ -9,43 +9,14 @@ spec:
         moduleName: kmm-ci-a 
 
       kernelMappings:
-        - literal: 5.4.196-108.356.amzn2.x86_64
-          containerImage: <aws_account_number>.dkr.ecr.<aws_region>.amazonaws.com/kmmo-lab:kmm-2.2
+        - regexp: '^.+.amzn2'
```
Comment (qbarrand) [inline docs/labs/aws/kmm-kmod.yaml:12]:
```suggestion
        - regexp: '^.+-amzn2$' 
```

File: docs/labs/gke/gke-ubuntu-nodes.yaml [hunk 4]
```diff
@@ -9,38 +9,15 @@ spec:
         moduleName: kmm-ci-a 
 
       kernelMappings:
-        - literal: 5.4.0-1078-gke 
+        - regexp: '^.+.gke'
```
Comment (qbarrand) [inline docs/labs/gke/gke-ubuntu-nodes.yaml:12]:
```suggestion
        - regexp: '^.+-gke$'
```

File: docs/labs/ubuntu/kmm-kmod.yaml [hunk 5]
```diff
@@ -9,37 +9,11 @@ spec:
         moduleName: kmm-ci-a 
 
       kernelMappings:
-        - literal: 5.15.0-43-generic
+        - regexp: '^.+.generic'
```
Comment (qbarrand) [inline docs/labs/ubuntu/kmm-kmod.yaml:12]:
```suggestion
        - regexp: '^.+-generic$'
```

File: docs/labs/aws/kmm-kmod.yaml (hunk 3 above)
Comment (enriquebelarte) [reply docs/labs/aws/kmm-kmod.yaml:12]: This won't be modified as regexp would not match with kernel version string.

**Pull request merged**